x/concentrated-liquidity/range_test.go

```diff
@@ -149,7 +149,8 @@ func (s *KeeperTestSuite) setupRangesAndAssertInvariants(pool types.Concentrated
 
 	// Ensure the pool balance is exactly equal to the assets added + amount swapped in - amount swapped out
 	poolAssets := s.App.BankKeeper.GetAllBalances(s.Ctx, pool.GetAddress())
-	s.Require().Equal(totalAssets, poolAssets)
+	poolSpreadRewards := s.App.BankKeeper.GetAllBalances(s.Ctx, pool.GetSpreadRewardsAddress())
+	s.Require().Equal(totalAssets, poolAssets.Add(poolSpreadRewards...))
```

Comment:
Do we need to add the pool's incentive address here as well?

Reply:
I don't believe so – the reason we have to add spread rewards is because the tracking we do wrt swaps is inclusive of fees (we add `tokenIn` and subtract `tokenOut` from `totalAssets` after each swap). Incentives aren't part of this equation